apps/storybook/src/SelectionTool.stories.tsx

```diff
@@ -17,230 +12,215 @@ import {
 
-  if (selectionModifierKey === panModifierKey) {
     return (
-      <p style={{ margin: '1rem', color: 'darkred' }}>
-        Pan and selection modifier keys cannot both be{' '}
-        <code>{panModifierKey || 'undefined'}</code>
-      </p>
-    );
-  }
-
-  return (
-    <VisCanvas
-      title={getTitleForSelection(activeSelection?.data)}
-      abscissaConfig={{ visDomain: [-10, 0], showGrid: true }}
-      ordinateConfig={{ visDomain: [50, 100], showGrid: true }}
-    >
-      <Pan modifierKey={panModifierKey} />
-      <Zoom />
-      <ResetZoomButton />
-
-      <SelectionTool
-        modifierKey={selectionModifierKey}
-        onSelectionChange={setActiveSelection}
-        onSelectionEnd={() => setActiveSelection(undefined)}
-        {...toolProps}
+      <VisCanvas
+        title={getTitleForSelection(activeSelection?.data)}
+        abscissaConfig={{ visDomain: [-10, 0], showGrid: true }}
+        ordinateConfig={{ visDomain: [50, 100], showGrid: true }}
       >
-        {({ html: htmlSelection }, _, isValid) => (
-          <SvgElement>
-            <SvgRect
-              coords={htmlSelection}
-              fill={isValid ? 'teal' : 'orangered'}
-              fillOpacity={0.5}
-              stroke={isValid ? 'teal' : 'orangered'}
-              strokeWidth={2}
-              strokePosition="inside"
-            />
-          </SvgElement>
-        )}
-      </SelectionTool>
-    </VisCanvas>
-  );
-};
-
-export const Rectangle = Template.bind({});
-
-export const WithModifierKey = Template.bind({});
-WithModifierKey.args = {
-  selectionModifierKey: 'Control',
-  panModifierKey: undefined,
-};
-
-export const WithValidation = Template.bind({});
-WithValidation.args = {
-  validate: ({ html }) => Box.fromPoints(...html).hasMinSize(100),
-};
-
-export const PersistedDataSelection: Story = () => {
-  const [persistedDataSelection, setPersistedDataSelection] = useState<Rect>();
-
-  return (
-    <VisCanvas
-      title={getTitleForSelection(persistedDataSelection)}
-      abscissaConfig={{ visDomain: [-10, 0], showGrid: true }}
-      ordinateConfig={{ visDomain: [50, 100], showGrid: true }}
-    >
-      <Pan modifierKey="Control" />
-      <Zoom />
-      <ResetZoomButton />
-
-      <SelectionTool
-        validate={({ html }) => Box.fromPoints(...html).hasMinSize(50)}
-        onSelectionStart={() => setPersistedDataSelection(undefined)}
-        onValidSelection={({ data }) => setPersistedDataSelection(data)}
-      >
-        {({ html: htmlSelection }, _, isValid) => (
-          <SvgElement>
-            <SvgRect
-              coords={htmlSelection}
-              fill={isValid ? 'teal' : 'orangered'}
-              fillOpacity="0.3"
-            />
-          </SvgElement>
-        )}
-      </SelectionTool>
-
-      {persistedDataSelection && (
-        <DataToHtml points={persistedDataSelection}>
-          {(...htmlSelection) => (
+        <Pan modifierKey={modifierKey?.length === 0 ? 'Control' : undefined} />
```

Comment:
The pan modifier key is now computing dynamically like in other stories. This allows the args/controls to match the real props of the `Pan` component.